### ISSUE TITLE
Info Widget or Info-Feature

### ISSUE THREAD

**Issue by superfunk2000** [Lissy93/dashy#229]
Hello Lissy93,
thank you very much for this great dashy! I replaced Homer with it and I am very satisfied.

Widgets are planned soon, is there an info widget included?
How I imagine it exactly:
If a certain text file exists, its content is displayed as info on Dashy. After confirming the information, the file will be deleted. In addition to the information, a heading and a (Font Awsom) icon can also be specified in the content of the text file.
Maybe like this:
 
```
Icon: "fas fa-water"
header: "Warning"
info: "Water sensor pump has triggered"
```

Greetings from Germany
Superfunk2000

**Comment by Lissy93** [comment]
Hello! 

So yeah, widgets are planned to be implemented before V2, an there will definitely be an info widget. I hadn't yet thought about how I would implement that specific one, but I like your idea. I will keep this issue open, and update you here once it's implemented :)

Also, thank you so much for your kind support via GitHub Sponsors, it's very very much appreciated <3  
And sorry for my mis-behaving bot!

**Comment by superfunk2000** [comment]
Hello Lissy,

after the bot closed my issue, I put the post under DIscussions. So it is now available twice. You can delete either one.

Many greetings
Superfunk 2000

**Comment by Lissy93** [comment]
Hiya @superfunk2000 

Just an update on this, beta support for widgets was merged in #382, if you update to V 1.9.4 you'll be able to use them. See the [Widget Docs](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md) for a list of pre-built widgets and usage guides.

Hope that helps :)
